CHANGELOG.md

```diff
@@ -33,6 +33,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
 ### Added
 
+- added class instances:
```

Comment:
This is not something the user can understand if not very familiar with CTL. Please describe the motivation for the change, and not the implementation. I think inserting a permanent link to the example code would be helpful too. Don't mention the tests (they are mandatory anyway).